index.js

```diff
@@ -6,7 +6,7 @@ class MethodRegistry {
 
   constructor (opts = {}) {
     this.provider = opts.provider ||
-      new Eth.HttpProvider('https://mainnet.infura.io/eth-contract-registry')
```

Comment:
This change is not needed, correct?

Reply:
not needed because of this: const registry = new MethodRegistry({ provider: global.ethereumProvider }) ?
or something else?
i'd still say we should update so any other consumers of the package utilize the correct endpoint